core/primitives/src/version.rs

```diff
@@ -111,6 +111,17 @@ pub enum ProtocolFeature {
     ChunkOnlyProducers,
     #[cfg(feature = "protocol_feature_lower_data_receipt_cost")]
     LowerDataReceiptCost,
+    /// This feature switch our WASM engine implementation from wasmer 0.* to
+    /// wasmer 2.*, brining better performance and reliability.
+    ///
+    /// The implementations should be sufficiently similar for this to not be a
+    /// protocol upgrade, but we concervatively do a protocol upgrade to be on
+    /// the safe side.
+    ///
+    /// Although wasmer2 is faster, we don't change fees with this protocol
+    /// version -- we can safely do that in a separate step.
```

Comment:
Yay!

Reply:
Also note that on the weekend we completed full history playback with Wasmer2, so it seems to work fine.